bclconvert.wdl

```diff
@@ -22,33 +22,39 @@ struct FastqCollection {
 
 
 workflow bclconvert {
+
   input {
     String runDirectory
     String runName
-    Array[Sample] samples
+    Array[Sample] samples 
+    Array[Int] lanes = []
     String? basesMask
     Int mismatches = 1
     String modules
     Int timeout = 40
-
   }
+
   parameter_meta {
-    runDirectory: "The path to the instrument's output directory."
+    runDirectory: {
+      description: "Illumina run directory (e.g. /path/to/191219_M00000_0001_000000000-ABCDE).",
+      vidarr_type: "directory"
+    }
     runName: "The name of the run, this will be used for the output folder and as a file prefix"
     samples: "array of Samples, that will includes names and barcodes"
+    lanes: "Extract reads only for specified lanes"
     basesMask: "An Illumina bases mask string to use. If absent, the one written by the instrument will be used."
     mismatches: "Number of mismatches to allow in the barcodes (usually, 1)"
     timeout: "The maximum number of hours this workflow can run for."
     modules: "Modules to run on hpc"
   }
 
   output {
-    File reports = bclconvert.reports
```

Comment:
The reports have important information such as read counts, why are they being dropped?

Reply:
I was under impression that Larry removed it, but it turns out it was one of the outputs when I started working on bclconvert. I put it back

Reply:
We don't want the reports to be output for each bclconvert workflow run. Correct the reports have information we need, but this information should be extracted from bclconvert.reports and output as a output struct/Pair<File,Map<String,String>>/vidarr file labels

Reply:
The thing that Michael suggests is already in place. Read counts are attached to fastq output, so I am going to revert my changes and remove report from the outputs